crypto/bls12381_utils.go

```diff
@@ -100,6 +100,17 @@ func (p *pointG2) equals(other *pointG2) bool {
 	return C.ep2_cmp((*C.ep2_st)(p), (*C.ep2_st)(other)) == valid
 }
 
+// Comparison to zero in Zr.
+// Scalar must be already reduced modulo r
+func (x *scalar) isZero() bool {
+	return C.bn_is_zero((*C.bn_st)(x)) == 1
+}
+
+// Comparison to point at infinity in G2.
+func (p *pointG2) isInfinity() bool {
+	return C.ep2_is_infty((*C.ep2_st)(p)) == 1
+}
+
```

Comment:
do we have some unit tests for these, it looks a bit complex logic to not have unit tests.

Reply:
Usually, there aren't specific unit tests for non exported APIs in the crypto module. However, these internal tools are tested implicitly using high level scenarios and edge cases. If there is a problem with them, many exported API tests would fail. 

I've also added a test case to cover a code path that I noticed it may be non covered so far. https://github.com/onflow/flow-go/pull/3421/commits/d54c7ee077adcccc6265938a14a116ec3ff1ac8e